lib/web_ui/lib/src/engine/semantics/scrollable.dart

```diff
@@ -28,11 +28,31 @@ import 'semantics.dart';
 /// viewport "scrollTop" may take positive values.
 class Scrollable extends RoleManager {
   Scrollable(SemanticsObject semanticsObject)
-      : super(Role.scrollable, semanticsObject);
+      : super(Role.scrollable, semanticsObject) {
+        _scrollOverflowElement.style
+        ..position = 'absolute'
+        ..transformOrigin = '0 0 0'
+        // Ignore pointer events since this is a dummy element.
+        ..pointerEvents = 'none';
```

Comment:
The `..` statements are continuation of the same line, so they should be indented. One easy way to format code is using "Format Selection" in VSCode (or Cmd + K + F). It doesn't always produce formatting compliant with Flutter's style guide, but it's usually a good first pass.